Sources/MulticastDelegate.swift

```diff
@@ -114,13 +111,10 @@ public func +=<T>(left: MulticastDelegate<T>, right: T) -> MulticastDelegate<T>
  *
  *  - parameter left:   The multicast delegate
  *  - parameter right:  The delegate to be removed
- *
- *  - returns: The `MulticastDelegate` instance with the `right` delegate parameter removed
  */
-public func -=<T>(left: MulticastDelegate<T>, right: T) -> MulticastDelegate<T> {
+public func -=<T>(left: MulticastDelegate<T>, right: T) {
```

Comment:
This warning is erroneous and should be ignored... Swift requires that there _isn't_ any space  around this...